src/mimeparser.rs

```diff
@@ -3299,4 +3303,36 @@ Message.
 
         Ok(())
     }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn test_receive_todo() -> Result<()> {
```

Comment:
i do not understand this function name, maybe `test_receive_eml()` is clearer?

```suggestion
    async fn test_receive_eml() -> Result<()> {
```

Reply:
Oops, yeah, "todo" meant "name this function later"